views/navigation.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
add only the content that has to be merged into the navigation. not the complete navigation